Calculator.Lonchanick/Calculator.Lonchanick/Operations.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Calculator.Lonchanick
+{
+    public class Operations
+    {
+        public double Operando1 { get; set; }
+        public double Operando2 { get; set; }
+        public double Result{ get; set;}
+        public string Operation { get; set; }
+        public Operations(double op1, double op2, double result, string operation)
+        {
+            this.Operando1 = op1;
```

Comment:
You don't need "this". The compiler takes care of it for you :D 
Usually the compiler will tell you these things when you hover over faded keywords (even though I'll concede you can barely see that "this" is faded here)

![image](https://user-images.githubusercontent.com/108711476/211710306-c06fbd87-19a4-4f2a-95dc-acdf0264002c.png)